packages/lodestar/src/sync/utils/sync.ts

```diff
@@ -0,0 +1,39 @@
+import {IReputation} from "../IReputation";
+import {Checkpoint, Epoch, Slot} from "@chainsafe/eth2.0-types";
+
+export function getTargetEpoch(peers: IReputation[], currentCheckPoint: Checkpoint): Epoch {
+  const peersWithHigherFinalizedEpoch = peers.filter(peer => {
+    if(!peer.latestHello) {
```

Comment:
How recent are the hello's?

Reply:
not recent, I don't think we are refreshing hellos, but I would do this in another PR as it's needed in regular sync as well. Possible solution, every SECONDS_PER_SLOT go trough all peers and refresh hellos.

Reply:
Yep!

Intitially i was thinking they would be new because of startup, but the more i thought about it, we could lag at any point need to resync and our hello's our totally out of wack

Another thing to think about is when we actually start syncing. Do we have an ideal `PEER_THRESHOLD` where we hold off on syncing until we have an ideal amount of peers

Reply:
I put 1 for now, as we will test it against single node most of the time. But I will add that as cli option in this PR so user can configure (I think you can do this in current eth1 nodes).

I left peers as reference to peers in sync module so sync can refresh peer data (and it will reflect in regular and initial sync) without some complicated passing on.